code/modules/mob/living/carbon/human/species/synthetic.dm

```diff
@@ -89,6 +90,7 @@
 	stun_reduction = 3.5
 
 	inherent_verbs = list(
```

Comment:
You can just remove this list and it will no longer override the parent list and instead inherit from the list on line 45

Reply:
Should be done and good now